index.js

```diff
@@ -0,0 +1,396 @@
+const inventory = [
```

Comment:
**Negeer deze pull request volledig!**
Dit is de link naar de juiste pull request!
https://github.com/wil004/Tech-It-Easy-Novi/pull/1/commits/8d73d92e228544079c8a264d8159b50b93edb979